app/controllers/organizations/staff/people_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Organizations::Staff::PeopleController < Organizations::BaseController
+  skip_before_action :verify_and_set_current_person
+  skip_verify_authorized only: %i[new create]
+  before_action :validate_person_does_not_exist
+  
+  def new
+    @person = Person.new
+  end
+
+  def create
+    @person = Person.new(person_params)
+    if @person.save
+      Current.user.add_role(:adopter, Current.organization)
```

Comment:
Yes this makes sense to only allow someone to create a person with an adopter role, since staff and fosterers need to be invited.